hw/top_earlgrey/jtag_id_pkg.core

```diff
@@ -0,0 +1,19 @@
+CAPI=2:
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+name: "lowrisc:constants:jtag_id_pkg"
+description: "jtag id for top_earlgrey"
+filesets:
+  files_rtl:
+    depend:
+      - "fileset_partner  ? (partner:constants:jtag_id_pkg)"
```

Comment:
i'm doing it this way so that any other block that needs to reference this id doesn't have to worry about the `fileset_partner`.

we should really do the same for `ast_pkg`

Reply:
Looks good.
We're ready on our side, so you can merge.
Also, there's a foundry CL that needs to be approved as well to support this on our side.

thanks